### PR TITLE
Fixed FEC export

### PR DESCRIPTION
**Pull request by Zank94**

### CODE REVIEW

File: app/concepts/fec/exporter/xml.rb [hunk 1]
```diff
@@ -60,7 +60,7 @@ def build(journals)
             end
           end
         end
-        builder.to_xml
+        builder.to_xml.gsub("&#8217;", "'")
```
Comment (piotaixr) [inline app/concepts/fec/exporter/xml.rb]: https://www.rubydoc.info/github/sparklemotion/nokogiri/Nokogiri/XML/Node#to_xml-instance_method
You should set the encoding in the to_xml method :-) U+8217 is the Unicode code for the `'` character.

Comment (Zank94) [reply app/concepts/fec/exporter/xml.rb]: `builder.to_xml(encoding: 'ISO-8859-15')`
doesn't work either, unfortunately

File: app/jobs/fec_export_job.rb [hunk 2]
```diff
@@ -11,7 +11,11 @@ def perform(financial_year, fiscal_position, interval, user)
         fec = FEC::Exporter::XML.new(financial_year, fiscal_position, period.first, period.last)
         file_path = Ekylibre::Tenant.private_directory.join('tmp', "#{filename}")
         FileUtils.mkdir_p(file_path.dirname)
-        File.write(file_path, fec.generate)
+
+        File.open(file_path, "wb:ISO-8859-15") do |fout|
```
Comment (Aquaj) [inline app/jobs/fec_export_job.rb]: 😮

Comment (Aquaj) [reply app/jobs/fec_export_job.rb]: (I didn't know you could do that)

**Pull request merged**